src/main/python/ayab/ayab.py

```diff
@@ -48,6 +52,12 @@
 
 MACHINE_WIDTH = 200
 
+# TODO move to generic configuration
+MACHINE_WIDTH = 200
```

Comment:
Duplicate of line 53

Reply:
Thanks. Eventually I'm going to move all the constants into a single module as you suggested.